content/en/docs/concepts/services-networking/service.md

```diff
@@ -678,6 +665,33 @@ spec:
       nodePort: 30007
 ```
 
+#### Custom IP address configuration for `type: NodePort` Services {#service-nodeport-custom-listen-address}
+
+You can set up nodes in your cluster to use a particular IP address for serving node port
+services. You might want to do this if each node is connected to multiple networks (for example:
+one network for application traffic, and another network for traffic between nodes and the
+control plane).
+
+If you want to specify particular IP(s) to proxy the port, you can set the
```

Comment:
Again, I think "IP addresses" would be better.

Reply:
I'll change it, but do bear in mind

> Watch out for pull requests that move content. If an author renames a page or combines two pages, we (Kubernetes SIG Docs) usually avoid asking that author to fix every grammar or spelling nit that we could spot within that moved content.

from https://kubernetes.io/docs/contribute/review/reviewing-prs/

When we review PRs where people have moved content, we try to go easy on the author if they have moved content around and the original text is not perfect.